### PR TITLE
fix(ingest): emit status aspect for entities ingested from okta and azure_ad

### PR DESCRIPTION
**Pull request by aditya-radhakrishnan**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [x] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py [hunk 1]
```diff
@@ -299,17 +300,27 @@ def get_workunits(self) -> Iterable[MetadataWorkUnit]:
                         aspectName="origin",
                         aspect=OriginClass(OriginTypeClass.EXTERNAL, "AZURE_AD"),
                     )
-                    group_origin_wu_id = (
-                        f"group-origin-{group_count + 1}"
-                        if self.config.mask_group_id
-                        else datahub_corp_group_snapshot.urn
-                    )
+                    group_origin_wu_id = f"group-origin-{group_count + 1 if self.config.mask_group_id else datahub_corp_group_snapshot.urn}"
```
Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py:303]: interesting change. lint did this i guess

File: metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py [hunk 2]
```diff
@@ -442,15 +449,23 @@ def ingest_ad_users(
                 aspectName="origin",
                 aspect=OriginClass(OriginTypeClass.EXTERNAL, "AZURE_AD"),
             )
-            user_origin_wu_id = (
-                f"user-origin-{user_count + 1}"
-                if self.config.mask_user_id
-                else datahub_corp_user_snapshot.urn
-            )
+            user_origin_wu_id = f"user-origin-{user_count + 1 if self.config.mask_user_id else datahub_corp_user_snapshot.urn}"
```
Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py:452]: linter?

File: metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py [hunk 3]
```diff
@@ -426,11 +437,7 @@ def ingest_ad_users(
                 assert datahub_group_membership
                 datahub_corp_user_snapshot.aspects.append(datahub_group_membership)
             mce = MetadataChangeEvent(proposedSnapshot=datahub_corp_user_snapshot)
-            wu_id = (
-                f"user-{user_count + 1}"
-                if self.config.mask_user_id
-                else datahub_corp_user_snapshot.urn
-            )
+            wu_id = f"user-snapshot-{user_count + 1 if self.config.mask_user_id else datahub_corp_user_snapshot.urn}"
```
Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py:440]: linter?

File: metadata-ingestion/src/datahub/ingestion/source/identity/okta.py [hunk 4]
```diff
@@ -386,17 +389,27 @@ def get_workunits(self) -> Iterable[MetadataWorkUnit]:
                     aspectName="origin",
                     aspect=OriginClass(OriginTypeClass.EXTERNAL, "OKTA"),
                 )
-                user_origin_wu_id = (
-                    f"user-origin-{user_count + 1}"
-                    if self.config.mask_user_id
-                    else datahub_corp_user_snapshot.urn
-                )
+                user_origin_wu_id = f"user-origin-{user_count + 1 if self.config.mask_user_id else datahub_corp_user_snapshot.urn}"
```
Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/ingestion/source/identity/okta.py:392]: linter?

File: metadata-ingestion/tests/integration/azure_ad/azure_ad_mces_golden_default_config.json [hunk 5]
```diff
@@ -12,7 +12,8 @@
                         "admins": [],
                         "members": [],
                         "groups": [],
-                        "description": null
+                        "description": null,
+                        "slack": null
```
Comment (jjoyce0510) [inline metadata-ingestion/tests/integration/azure_ad/azure_ad_mces_golden_default_config.json:16]: nice

**Review by jjoyce0510 — APPROVED** [review]
Overall LGTM. Thank you so much, Aditya!

File: metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py (hunk 1 above)
Comment (aditya-radhakrishnan) [reply metadata-ingestion/src/datahub/ingestion/source/identity/azure_ad.py:303]: Resolved offline! Wanted to make the WU ID more clear for unmasked cases

**Pull request merged**